.github/workflows/connectors_version_increment.yml

```diff
@@ -0,0 +1,71 @@
+name: Connectors Version Bump
```

Comment:
I find the name misleading: it is not bumping the version, it is checking if the version was incremented